services/core-api/tests/now_application_factories.py

```diff
@@ -454,9 +454,14 @@ class Meta:
 
     class Params:
         mine = factory.SubFactory('tests.factories.MineFactory')
-        submission_only = factory.Trait(now_application=None, now_application_id=None, mms_cid=None)
+        submission_only = factory.Trait(
+            now_application=None,
+            now_application_id=None,
+            mms_cid=None,
+            application_type_code=factory.LazyFunction(RandomApplicationTypeCode))
```

Comment:
We want a random code for submission_only? I think all submissions should be "NOW" instead?

Reply:
done